test/integration/arbitrum-mainnet/Loans.fork.t.sol

```diff
@@ -449,6 +457,14 @@ contract LoansForkTest is LoansTestBase {
         assertGt(protocolFee, 0);
     }
 
+    function getProviderProtocolFeeByUnderlying() internal view returns (uint protocolFee) {
```

Comment:
didn't I [change this in `develop`](https://github.com/CollarNetworks/protocol-core/blob/develop/test/integration/arbitrum-mainnet/Loans.fork.t.sol#L230-L236)? This calculation is incorrect (and failed the tests on my end) due to assuming no slippage for the swap.

Was it changed back when merging from develop somehow?